src/Microsoft.Azure.WebJobs.Host/Common/GenericBinder.cs

```diff
@@ -10,17 +10,39 @@
 using Microsoft.Azure.WebJobs.Host.Protocols;
 using Microsoft.Azure.WebJobs.Host.Triggers;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
 {
-    // Helpers for doing general-purpose bindings. 
-    internal class GenericBinder
+    /// <summary>
+    /// Helpers for creating bindings to common patterns such as Messaging or Streams.
+    /// </summary>
+    public static class GenericBinder
```

Comment:
sounds good. 

---
In reply to: [54931702](https://github.com/Azure/azure-webjobs-sdk/pull/643#discussion_r54931702) [](ancestors = 54931702)